packages/core/parcel/package.json

```diff
@@ -14,9 +14,13 @@
     "type": "git",
     "url": "https://github.com/parcel-bundler/parcel.git"
   },
-  "bin": "src/bin.js",
+  "bin": "bin/parcel.js",
   "main": "lib/bin.js",
   "source": "src/bin.js",
+  "scripts": {
+    "prepublishOnly": "./ensure-no-dev-lib.sh",
```

Comment:
Could you just have this on the root `prebublishOnly` script and check the entire repo?

Reply:
I don't know if the root hook is called when children are published, but I'd guess not?

Reply:
AFAIK the entire publish pipeline currently relies on the root script to build the packages cc @mattcompiles

Reply:
I think the root script will build the packages, but at the npm/yarn publish level this would be run by the package manager

Reply:
Having per package scripts is a bit nicer because you can run them separately with built-in tooling (+ possibly supercharge with turbo-repo)

Reply:
I personally don't like the repetition, but it's not a big thing. /nit you could at least share the script in root `scripts` folder since it's all the same

Reply:
Yeah; that's okay. 

So maybe I can symlink these files. Let me try

Reply:
There is no repetition anymore; only symlinks